todo_app/data/classes.py

```diff
@@ -0,0 +1,6 @@
+class to_do_item:
```

Comment:
Minor - I think this file makes more sense to be called `todo_item.py`. If we add more classes in future we probably want them to be in separate files.